testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -274,8 +274,7 @@ private static String createQueryDebuggingSummary(BasicQueryInfo basicQueryInfo,
         }
     }
 
-    // TODO @Test - Temporarily disabled to avoid test classes running twice. Re-enable once all tests migrated to JUnit.
-    @ReportBadTestAnnotations.Suppress
+    @Test
```

Comment:
Some ATQF subclasses may still be testng?

Reply:
I looked for `testng` occurences in the codebase and did not find any relevant. Hope did not miss anything.